types.ts

```diff
@@ -0,0 +1,28 @@
+export type TableStreamItem = {
```

Comment:
Add comment to explain that this provides a missing type for a data structure from the Architect inventory.